thalwegPlotsMEDIAN.py

```diff
@@ -0,0 +1,418 @@
+                axes[i].set_facecolor('xkcd:white')
+                axes[i].grid(linestyle="-", color='lightgrey', alpha=0.4, axis='y')
+                #rect = Rectangle((0, 0), 1, 1, transform=fig.transFigure, color="darkgrey", fill=False, lw=1)
+                #fig.patches.append(rect)
+        
+        #plot is sved        
+        outFile = ('violinplots_%s_%s' %(namePlot,split))
+
+    #if only one violin is plotted and not splitted by the split attribute    
+    else:
+        colPalette = ['#a1cae2', '#0b3f5e', '#3b93ea','#de6326','#d17547','#0fbddb', '#1c5761']
+        fig = plt.figure(figsize =(6*int(len(colList)),8))
+        ax3 = plt.subplot(111)
+        sns.stripplot(data=dbData[colList], jitter=True, color ='grey', alpha=0.5, size=2,  zorder=1)
+        sn = sns.violinplot(data=dbData[colList], palette= colPalette, fill=False, cut=0)
+        
+        if ylim != 0:
+            sn.set(ylim=ylim)
+        
+        ax3.tick_params(labelsize=16)
+        
+        #calculation of statistical values, adds them to plot
+        for idx, col in enumerate(colList):
+            mean_value = dbData[col].median()
+            quantile = dbData[col].quantile(0.75)
+            x_position = idx 
+            ax3.text(x_position, mean_value, f'{mean_value:.1f}', ha='center', va='center', color='black', fontsize=22)
+            
+        if renameTitle == []:
+            ax3.set_title('Distribution of %s for %d events' % (namePlot, len(dbData[colList[0]])), fontsize=20)
+        else:
+            ax3.set_title(renameTitle[0] + ' for %d events' %len(dbData[colList[0]]),fontsize=20)
+        
+        if renameY == []:
+            ax3.set_ylabel(namePlot)
+        else:
+            ax3.set_ylabel(renameY[0], fontsize = 20)
+            
+        if renameX == []:
+            ax3.set_xlabel(namePlot)
+        else:
+            ax3.set_xticklabels(renameX, fontsize=16)
+        
+        #ax3.set_xlabel('thalwegs', fontsize = 20)
+        sns.set(style="whitegrid")
+        ax3.tick_params(axis='both', labelsize=20)
+        outFile = ('violinplots_%s' % namePlot)
+    # save figure
+    plotPath = pU.saveAndOrPlot({'pathResult': outDir}, outFile, fig)
+    
+    return plotPath
+
+
+    
+def multiplePlots3 (plist, name, title, outdir):
+    ''' puts multiple plots next to each other and underneath each other, 
+        saves them as new plot and delets single plots'''
+    
+    nrows = 2
+    ncols = 2
+    
+    # Create a figure with 2 rows and 2 columns
+    fig, axes = plt.subplots(nrows=nrows, ncols=ncols, figsize=(15, 10))
+    fig.suptitle(title)
+    
+    # Loop over the plots and add them to the figure
+    for i, plotpath in enumerate(plist):
+        row = i // ncols  # Determine the row index
+        col = i % ncols   # Determine the column index
+        axes[row, col].imshow(plt.imread(plotpath))
+        axes[row, col].axis('off')  # Hide the axes for better display
+        os.remove(plotpath)
```

Comment:
shall we keep this?

Reply:
Nope. Or if it is wanted, it could be controlled via flag (similar to our main avaframe cfg)